docs/develop.md

```diff
@@ -100,3 +104,9 @@ $ api/build-tools/schema-sync.sh master
 # Using Apache APISIX's version
 $ api/build-tools/schema-sync.sh 2.0
 ```
+
+If you have custom plugin, please make sure your custom plugin is placed in the APISIX directory, and change the parameters of the execution script to the APISIX directory path, example:
+
+```sh
+$ api/build-tools/schema-sync.sh /usr/local/apisix
```

Comment:
that is not a good example, I think `/your/path/apisix` is better.

Reply:
Aha this doc is outdated, we have another in reviewing PR.